helper/DockerHelper.go

```diff
@@ -200,9 +200,15 @@ func BuildArtifact(ciRequest *CiRequest) (string, error) {
 	}
 	if ciBuildConfig.CiBuildType == SELF_DOCKERFILE_BUILD_TYPE || ciBuildConfig.CiBuildType == MANAGED_DOCKERFILE_BUILD_TYPE {
 		dockerBuild := "docker build "
+		if ciRequest.CacheInvalidate && ciRequest.IsPvcMounted {
+			dockerBuild = "docker build --no-cache"
```

Comment:
append no chache to dockerBuild instead of assigning value to it

Reply:
done